bootstrap.sh

```diff
@@ -25,8 +25,24 @@ else
     exit 1
 fi
 
+get_env_var_with_optional_default() {
+   local var_name=$1
+   local default_val=$2
+   eval "existing_value=\${$var_name}"
+    if [[ -n $existing_value ]]; then
+        echo "$existing_value"
+    elif [[ -n $default_val ]]; then
+        echo "$default_val"
+    fi
+}
+
+readonly MAVEN_CMD=$(get_env_var_with_optional_default VESPA_MAVEN_COMMAND mvn)
+readonly MAVEN_EXTRA_OPTS=$(get_env_var_with_optional_default VESPA_MAVEN_EXTRA_OPTS)
+echo "Using maven command: ${MAVEN_CMD}"
+echo "Using maven extra opts: ${MAVEN_EXTRA_OPTS}"
+
 mvn_install() {
-    mvn --quiet --batch-mode --no-snapshot-updates clean install -Dmaven.javadoc.skip=true "$@"
+    ${MAVEN_CMD} --quiet --batch-mode --no-snapshot-updates clean install -Dmaven.javadoc.skip=true ${MAVEN_EXTRA_OPTS} "$@"
```

Comment:
Consider removing `--quiet`, `--Dmaven.javadoc.skip=true` and `--batch-mode`.

Reply:
Good plan, but let's do it in a separate PR, and notify the team shortly afterwards.